code/modules/mob/living/carbon/xenomorph/abilities/praetorian/praetorian_powers.dm

```diff
@@ -12,50 +12,52 @@
 	if (!check_and_use_plasma_owner())
 		return
 
+	//X = xeno user, A = target atom
+	var/list/turf/target_turfs = getline2(X, A, include_from_atom = FALSE)
```

Comment:
then just rename the vars xeno_user and target_atom, why use abbreviations? I haven't looked at the file, maybe it's plagued with too many abbreviations.

Reply:
These vars are defined outside the lines I've actually changed. Again I can change them if needed.

Reply:
I see that, like I said I haven't looked in the file and maybe it's plagued with too many abbreviations and it's kind of outside the scope of this pr. Something hacky you could do, is just var/mob/living/carbon/xenomorph/xeno_user = A or whatever the path is and then the type path for the other var as well, but I must stress, that it's better to do it the right way.

Reply:
Ok, I took a look at the file. You should definitely change the var names for at least that proc that you're working in.

Reply:
Renamed these vars in latest commit.